src/commands/Info Commands/MdnCommand.ts

```diff
@@ -0,0 +1,48 @@
+import { MessageEmbed } from 'discord.js';
+import nodeFetch from 'node-fetch';
+import { RunFunction } from '../../interfaces/Command';
+import paginationEmbed from 'discord.js-pagination';
+
+export const run: RunFunction = async (client, message, args) => {
+	if (!args.length)
+		return message.channel.send(
+			'Invalid usage, provide a search term! `mdn <search input>`'
+		);
+
+	const pages = [];
+
+	nodeFetch(
+		`https://developer.mozilla.org/api/v1/search/en-US?q=${args.join(
+			' '
+		)}&locale=en-US&highlight=false`
+	)
+		.then((body) => body.json())
+		.then(async (result) => {
+			result.documents.forEach((document) => {
+				if (!document.slug.toLowerCase().includes('javascript')) {
+					return;
+				}
+
+				pages.push(
+					new MessageEmbed()
```

Comment:
use client.embed

Reply:
I can't use client.embed for the pages because the pagination package sets its own footers

Reply:
OK